### DISCUSSION TITLE
about select_sequential_consistency

### DISCUSSION THREAD

**Discussion by honobi** [ClickHouse/ClickHouse#66569] [Q&A]
What is the principle behind the select_sequential_consistency parameter? Does it force the current replica to receive the latest updates, or does it make the query use only the most recent replica?

**Comment by yiyang-shao** [comment]


**Reply by qoega** [reply]
This response looks as a LLM generated one and is completely incorrect.

**Comment by qoega** [comment]
select_sequential_consistency exists in 2 versions: OSS when insert_quorum is used and in Cloud version for SharedMergeTree engine.

For SMT version this setting basically requires replica that has query to read latest state from Keeper. It increases load on keeper.
